### ISSUE TITLE
impossible to compile tum_ardrone

### ISSUE THREAD

**Issue by Lek** [tum-vision/tum_ardrone#8]
When I build (cmd : rosmake), I have this errors

[ 77%] Built target drone_stateestimation
  /opt/ros/groovy/tum_ardrone/src/UINode/RosThread.cpp: In member function ‘void RosThread::navdataCb(ardrone_autonomy::NavdataConstPtr)’:
  /opt/ros/groovy/tum_ardrone/src/UINode/RosThread.cpp:88:24: error: ‘const struct ardrone_autonomy::Navdata_std::allocator<void >’ has no member named ‘motor1’
  /opt/ros/groovy/tum_ardrone/src/UINode/RosThread.cpp:89:24: error: ‘const struct ardrone_autonomy::Navdata_std::allocator<void >’ has no member named ‘motor2’
  /opt/ros/groovy/tum_ardrone/src/UINode/RosThread.cpp:90:24: error: ‘const struct ardrone_autonomy::Navdata_std::allocator<void >’ has no member named ‘motor3’
  /opt/ros/groovy/tum_ardrone/src/UINode/RosThread.cpp:91:24: error: ‘const struct ardrone_autonomy::Navdata_std::allocator<void >’ has no member named ‘motor4’
  make[4]: **\* [CMakeFiles/drone_gui.dir/src/UINode/RosThread.cpp.o] Error 1
  make[4]: Leaving directory `/opt/ros/groovy/tum_ardrone/build'
  make[3]: *** [CMakeFiles/drone_gui.dir/all] Error 2
  make[3]: Leaving directory`/opt/ros/groovy/tum_ardrone/build'
  make[2]: **\* [all] Error 2
  make[2]: Leaving directory `/opt/ros/groovy/tum_ardrone/build'
  make[1]: *** [all] Error 2
  make[1]: Leaving directory`/opt/ros/groovy/tum_ardrone'

Any Idea ?

**Comment by Lek** [comment]
Solution :
In the file Navata.msg in the folder ardrone_autonomy/msg
Add this lines : 
#motor commands (unit 0 to 255)
uint8 motor1
uint8 motor2
uint8 motor3
uint8 motor4

**Comment by lesire** [comment]
I guess you rely on the tum-vision/ardrone_autonomy package ?
I personally use the original AutonomyLab/ardrone_autonomy package, on both groovy and hydro without problem.

Regarding compilation of tum_ardrone, we are updating the sources to use catkin. Everything should be available within a few days... stay connected!

**Comment by mmyself** [comment]
i think it will be great if we only have the autonomyLab ardrone package from maniji and the tum_ardrone, so there will not be the double development.
And if i am right tum_ardrone now works well with the orig autonomy_ardrone

**Comment by lesire** [comment]
Yes it is, at least for groovy and hydro. I don't know about fuerte...

The doc must definitely be updated in that sense: I'll create an issue for that!

**Comment by lesire** [comment]
Could you check that everything is now OK with the new 'hydro-devel' branch, and then close or update this issue?
